apps/api/src/.ormconfig.ts

```diff
@@ -11,8 +11,8 @@ export const config: ConnectionOptions = {
   username: process.env.POSTGRES_USERNAME,
   password: process.env.POSTGRES_PASSWORD,
   database: process.env.POSTGRES_DATABASE,
-  entities: ['dist/**/*.entity.js'],
-  migrations: ['src/database/migrations/*.ts'],
+  entities: ['./**/*.entity.js'],
```

Comment:
are you sure this is working with docker?

Reply:
yes